src/components/FourOhFour/FourOhFour.jsx

```diff
@@ -3,8 +3,12 @@ import styles from './FourOhFour.css';
 
 export default () => (
   <div className={styles.container}>
-    <h1>404</h1>
-    <h2>Page not found.</h2>
-    :(
+    <img src="/assets/images/sad.gif" role="presentation" />
+    <div className={styles.message}>
+      The page you’re looking for can’t be found.
```

Comment:
consider putting the strings in `lang/en.json`.  We don't HAVE to do it now, but anything we do now we don't have to do later.